clarity-ext-scripts/clarity_ext_scripts/covid/rtpcr_analysis_service.py

```diff
@@ -64,18 +64,28 @@ def __init__(self, covid_reporter_key, internal_control_reporter_key):
 
     # TODO Later we might have to account for giving two trails over 38 as positive.
 
-    def _analyze_sample(self, sample):
+    # POS Control
+    # FAM=0 and VIC=0 => FAIL
+    # NEG Control
+    # FAM=0 and VIC=0 => NEG
+    # SAMPLE
+    # FAM=0 and VIC=0 => FAIL
+
+    def _analyze_sample(self, sample, is_neg_control=False):
```

Comment:
I don't like to have is_neg_control as an input arg here. But this doesn't matter today, as this fixes the thing.